### PR TITLE
Select tiflash with RR strategy

### PR DESCRIPTION
**Pull request by shiyuhang0**
### What problem does this PR solve? 

Select TiFlash store with RR strategy to avoid only one TiFlash works

### Test 

- [x] Test manually

### CODE REVIEW

File: tikv-client/src/test/java/com/pingcap/tikv/GrpcUtils.java [hunk 1]
```diff
@@ -21,6 +21,8 @@
 import com.pingcap.tikv.codec.Codec.BytesCodec;
 import com.pingcap.tikv.codec.CodecDataOutput;
 import java.util.Arrays;
+
```
Comment (Daemonxiao) [inline tikv-client/src/test/java/com/pingcap/tikv/GrpcUtils.java:24]:
```suggestion
```

**Review by zhangyangyu — DISMISSED** [review]

**Review by xuanyu66 — APPROVED** [review]

**Review by xuanyu66 — APPROVED** [review]

**Comment by shiyuhang0** [comment]
> /run-all-tests

success

**Pull request merged**